hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -140,6 +140,12 @@
 
   public static final String NO_OP_BOOTSTRAP_INDEX_CLASS = NoOpBootstrapIndex.class.getName();
 
+  public static final ConfigProperty<String> HOODIE_TABLE_NUM_BUCKETS = ConfigProperty
```

Comment:
why not reuse `hoodie.bucket.index.bucket.num`?

Reply:
done